src/cloud-api-adaptor/cmd/cloud-api-adaptor/main.go

```diff
@@ -121,8 +133,22 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 
 	fmt.Printf("%s: starting Cloud API Adaptor daemon for %q\n", programName, cloudName)
 
-	if !disableTLS {
-		cfg.serverConfig.TLSConfig = &tlsConfig
+	if secureComms {
+		err := kubemgr.InitKubeMgrInVivo()
+		if err != nil {
+			return nil, fmt.Errorf("secure comms failed to initialize KubeMgr: %w", err)
+		}
+
+		cfg.serverConfig.SecureComms = true
+		cfg.serverConfig.SecureCommsInbounds = secureCommsInbounds
+		cfg.serverConfig.SecureCommsOutbounds = secureCommsOutbounds
+		cfg.serverConfig.SecureCommsKbsAddress = secureCommsKbsAddr
+
+		cfg.serverConfig.AAKBCParams = AAKBC_PARAMS_DEFAULT
```

Comment:
s/AAKBC_PARAMS_DEFAULT/AA_KBC_PARAMS_DEFAULT
We use AA_KBC_PARAMS in many places to indicate the config. So the default will align with the existing naming

Reply:
changed to AA_KBC_PARAMS